cmd/curio/guidedsetup/guidedsetup.go

```diff
@@ -244,41 +246,127 @@ func complete(d *MigrationData) {
 	stepCompleted(d, d.T("Lotus-Miner to Curio Migration."))
 }
 
-var EnvFiles = []string{"/etc/curio.env", "./curio/curio.env", "~/config/curio.env"}
-
 func afterRan(d *MigrationData) {
-	// Write curio.env file.
-	// Inform users they need to copy this to /etc/curio.env or ~/.config/curio.env to run Curio.
-	places := append([]string{"/tmp/curio.env",
-		must.One(os.Getwd()) + "/curio.env",
-		must.One(os.UserHomeDir()) + "/curio.env"}, EnvFiles...)
-saveConfigFile:
-	_, where, err := (&promptui.Select{
-		Label:     d.T("Where should we save your database config file?"),
-		Items:     places,
-		Templates: d.selectTemplates,
-	}).Run()
+	// Determine the current user's shell
```

Comment:
I would definitely first ask if the user wants to put curio DB credentials in the shell rc file.

e.g. I have my dotfiles managed in git, and would be rather surprised to find new entries there without allowing/expecting them to be written there.